### ISSUE TITLE
[galera] Issue when trying to access the database through the LB

### ISSUE THREAD

**Issue by lkoenigsberger** [rancher/rancher-catalog#95]
I get the following error:
```
WSREP has not yet prepared node for application use
```
here's the output of SHOW STATUS LIKE 'wsrep%';
```
+------------------------------+--------------------------------------------+
| Variable_name                | Value                                      |
+------------------------------+--------------------------------------------+
| wsrep_local_state_uuid       | 936c150a-c458-11e5-80ef-c343596cc0e3       |
| wsrep_protocol_version       | 18446744073709551615                       |
| wsrep_last_committed         | 0                                          |
| wsrep_replicated             | 0                                          |
| wsrep_replicated_bytes       | 0                                          |
| wsrep_repl_keys              | 0                                          |
| wsrep_repl_keys_bytes        | 0                                          |
| wsrep_repl_data_bytes        | 0                                          |
| wsrep_repl_other_bytes       | 0                                          |
| wsrep_received               | 2                                          |
| wsrep_received_bytes         | 495                                        |
| wsrep_local_commits          | 0                                          |
| wsrep_local_cert_failures    | 0                                          |
| wsrep_local_replays          | 0                                          |
| wsrep_local_send_queue       | 0                                          |
| wsrep_local_send_queue_max   | 1                                          |
| wsrep_local_send_queue_min   | 0                                          |
| wsrep_local_send_queue_avg   | 0.000000                                   |
| wsrep_local_recv_queue       | 0                                          |
| wsrep_local_recv_queue_max   | 1                                          |
| wsrep_local_recv_queue_min   | 0                                          |
| wsrep_local_recv_queue_avg   | 0.000000                                   |
| wsrep_local_cached_downto    | 18446744073709551615                       |
| wsrep_flow_control_paused_ns | 0                                          |
| wsrep_flow_control_paused    | 0.000000                                   |
| wsrep_flow_control_sent      | 0                                          |
| wsrep_flow_control_recv      | 0                                          |
| wsrep_cert_deps_distance     | 0.000000                                   |
| wsrep_apply_oooe             | 0.000000                                   |
| wsrep_apply_oool             | 0.000000                                   |
| wsrep_apply_window           | 0.000000                                   |
| wsrep_commit_oooe            | 0.000000                                   |
| wsrep_commit_oool            | 0.000000                                   |
| wsrep_commit_window          | 0.000000                                   |
| wsrep_local_state            | 0                                          |
| wsrep_local_state_comment    | Initialized                                |
| wsrep_cert_index_size        | 0                                          |
| wsrep_causal_reads           | 0                                          |
| wsrep_cert_interval          | 0.000000                                   |
| wsrep_incoming_addresses     | unspecified,10.42.101.124:3306,unspecified |
| wsrep_evs_delayed            |                                            |
| wsrep_evs_evict_list         |                                            |
| wsrep_evs_repl_latency       | 0/0/0/0/0                                  |
| wsrep_evs_state              | OPERATIONAL                                |
| wsrep_gcomm_uuid             | 54508b17-c679-11e5-b552-ebe32d270566       |
| wsrep_cluster_conf_id        | 18446744073709551615                       |
| wsrep_cluster_size           | 3                                          |
| wsrep_cluster_state_uuid     | 936c150a-c458-11e5-80ef-c343596cc0e3       |
| wsrep_cluster_status         | non-Primary                                |
| wsrep_connected              | ON                                         |
| wsrep_local_bf_aborts        | 0                                          |
| wsrep_local_index            | 1                                          |
| wsrep_provider_name          | Galera                                     |
| wsrep_provider_vendor        | Codership Oy <info@codership.com>          |
| wsrep_provider_version       | 3.9(rXXXX)                                 |
| wsrep_ready                  | OFF                                        |
| wsrep_thread_count           | 2                                          |
+------------------------------+--------------------------------------------+
57 rows in set (0.02 sec)
```

**Comment by cloudnautique** [comment]
Was the cluster started then stopped? Are all of the nodes in this mode? It looks like split brain. Can all of the nodes communicate with each other?
These [docs](http://galeracluster.com/documentation-webpages/quorumreset.html#) might help.

**Comment by cloudnautique** [comment]
I think this was caused by #96, can you try deploying a new cluster?

**Comment by cloudnautique** [comment]
closing, was resolved by #96. please re-open if thats not the case.
